.gitignore

```diff
@@ -28,24 +28,8 @@ berkeley-function-call-leaderboard/eval_checker/tree-sitter-javascript
 berkeley-function-call-leaderboard/tree-sitter-java
 berkeley-function-call-leaderboard/tree-sitter-javascript
 
-# Ignore Evaluation Dataset files that we download from https://huggingface.co/datasets/gorilla-llm/Berkeley-Function-Calling-Leaderboard/tree/main
+# Ignore aggregated eval data (used for OSS models)
```

Comment:
Why so? Why not also include OSS evals?

Reply:
`eval_data_total.json` is generated by `eval_data_compilation.py`, which just compiles all the test files into one. There is no need to keep track of it, since it can (and should) be generated anew on each run (to accommodate any possible changes to the dataset.